src/libaktualizr/primary/aktualizr_test.cc

```diff
@@ -587,6 +587,9 @@ TEST(Aktualizr, InstallWithUpdates) {
   EXPECT_EQ(result.reports.size(), 0);
   conf.uptane.running_mode = RunningMode::kDownload;
   aktualizr.UptaneCycle();
+  EXPECT_NE(aktualizr.GetStoredTarget("primary_firmware.txt").get(), nullptr);
+  EXPECT_NE(aktualizr.GetStoredTarget("secondary_firmware.txt").get(), nullptr);
```

Comment:
I suggest also doing the negative test, as in, check that these are `nullptr` before running the `UptaneCycle`.

Reply:
Sure, makes sense.